tests/test_checkout_functionality.py

```diff
@@ -12,25 +17,40 @@
 class TestCheckoutProcess:
     def test_checkout_functionality(self, login_to_saucedemo, add_profuct_to_saucedome):
         checkout_button = driver.find_element(By.XPATH, '//button[@id="checkout"]')
+        assert checkout_button.is_displayed(), "The button is not visible"
         checkout_button.click()
 
         WebDriverWait(driver, 10).until(EC.visibility_of_element_located((By.ID, "checkout_info_container")))
         first_name_field = driver.find_element(By.XPATH, '//input[@id="first-name"]')
+        assert first_name_field.is_displayed(), "The first name is not visible"
         first_name_field.send_keys(FIRSTNAME)
```

Comment:
@Sv-24 you do not need to create static varible, just add under test first_name instead of FIRSTNAME

Reply:
@irenalavrinok done